thirdParty/cupla/example/CUDASamples/matrixMul/src/matrixMul.cpp

```diff
@@ -1,5 +1,4 @@
-/**
- * Copyright 1993-2014 NVIDIA Corporation.  All rights reserved.
+/* Copyright 1993-2014 NVIDIA Corporation.  All rights reserved.
```

Comment:
Is this Okay regarding the license?

Reply:
this is *cupla* source code. there is not reason to review it. here